webapp/src/components/MarketplacePage/Publication/Publication.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Header, Card, Button, Icon } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+import { publicationType } from 'components/types'
+import { txUtils } from 'decentraland-commons'
+import EtherscanLink from 'components/EtherscanLink'
+import numeral from 'numeral'
+import moment from 'moment'
+import './Publication.css'
+
+export default class Publication extends React.PureComponent {
+  static propTypes = {
+    publication: publicationType
+  }
+  render() {
+    const { publication } = this.props
+    let iconName = 'check circle outline'
+    let iconTooltip = 'Transaction confirmed'
+    if (publication.tx_status === txUtils.TRANSACTION_STATUS.pending) {
+      iconName = 'warning sign'
+      iconTooltip = 'Transaction pending'
+    } else if (publication.tx_status === txUtils.TRANSACTION_STATUS.failed) {
+      iconName = 'remove circle outline'
+      iconTooltip = 'Transaction failed'
+    }
+    const isExpired = publication.expires_at < Date.now()
+    return (
+      <Card className="Publication">
+        <Card.Content>
+          <Link to={`/${publication.x}/${publication.y}`}>
```

Comment:
To avoid future route renaming pains, this should be using `locations.parcelMapDetail(publication.x, publication.y)`